packages/flutter/lib/src/material/card_theme.dart

```diff
@@ -0,0 +1,130 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:ui' show lerpDouble;
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/widgets.dart';
+
+import 'theme.dart';
+
+/// Defines default property values for descendant [Card] widgets.
+///
+/// Descendant widgets obtain the current [CardTheme] object using
+/// `CardTheme.of(context)`. Instances of [CardTheme] can be
+/// customized with [CardTheme.copyWith].
+///
+/// Typically a [CardTheme] is specified as part of the overall [Theme]
+/// with [ThemeData.cardTheme].
+///
+/// All [CardTheme] properties are `null` by default. When null, the [Card]
+/// will use the values from [ThemeData] if they exist, otherwise it will
+/// provide its own defaults.
+///
+/// See also:
+///
+///  * [ThemeData], which describes the overall theme information for the
+///    application.
+class CardTheme extends Diagnosticable {
+
+  /// Creates a theme that can be used for [ThemeData.cardTheme].
+  const CardTheme({
+    this.clipBehavior,
+    this.color,
+    this.elevation,
+    this.margin,
+    this.shape,
+  });
+
+  /// Default value for [Card.clipBehavior].
+  final Clip clipBehavior;
+
+  /// Default value for [Card.color].
+  ///
+  /// If null, [Card] uses [ThemeData.cardColor].
```

Comment:
It's nice that the default value is reiterated here. Why not do the same for the other properties (even though they're constant values rather than ThemeData values)?

Reply:
done